.github/workflows/create-lint-wf.yml

```diff
@@ -1,5 +1,11 @@
 name: Create a pipeline and run nf-core linting
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - dev
```

Comment:
I would add `master`, too, possibly?

Reply:
```
  release:
    types: [published]
```
Handles that in theory.

Reply:
yeah, master might be nice 👍🏻  Won't happen often but means we get tests before release..